WebPhapp/client/src/App/pages/PrescriptionAdd.js

```diff
@@ -69,55 +69,141 @@ class PrescriptionAdd extends Component {
+                                <a className="dropdown-item" href="#">Something else here</a>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <br/>
 
-          <label>
-            Patient ID
-          </label>
-          <input
-            type="p"
-            value={this.state.patientID}
-            onChange={this.onKeyDownPatientID}
-          />
-
-          <label>
-            &nbsp; Drug&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.drugID}
-            onChange={this.onKeyDownDrugID}
-          />
-          <label>
-            &nbsp; Quantity&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.quantity}
-            onChange={this.onKeyDownQuantity}
-          />
-          <label>
-            &nbsp; Days Valid&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.daysValid}
-            onChange={this.onKeyDownDaysValid}
-          />
-          <label>
-            &nbsp; Dispensor ID&nbsp;
-          </label>
-          <input
-            type="p"
-            value={this.state.dispensorID}
-            onChange={this.onKeyDownDispensorID}
-          />
+        <form>
+          <div className="row">
+            <div className="col-md-6 center">
+              <div className="form-group">
+                <input 
+                type="p" 
+                className="form-control" 
+                placeholder="Patient ID"
+                // value={this.state.patientID}
+                onChange={this.onKeyDownPatientID}/> 
+              </div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-md-6 center">
+              <div className="form-group">
+                <input 
+                type="p" 
+                className="form-control" 
+                placeholder="Drug ID"
+                // value={this.state.drugtID}
+                onChange={this.onKeyDownDrugID}/> 
+              </div>
+            </div>
+          </div> 
+          <div className="row">
+            <div className="col-md-6 center">
+              <div className="form-group">
+                <input 
+                type="p" 
+                className="form-control" 
+                placeholder="Dispenser ID"
+                // value={this.state.dispensorID}
+                onChange={this.onKeyDownDispensorID}/> 
+              </div>
+            </div>
+          </div> 
+          <div className="row">
+            <div className="col-md-6 center">
+              <div className="form-group">
+                <input 
+                type="p" 
+                className="form-control" 
+                placeholder="Quantity"
+                // value={this.state.quantity}
```

Comment:
Why are these commented out?

Reply:
these were commented out because each attribute used to have a label that corresponded to a textfield. I removed the label and put the label inside each textfield by assigning each attribute a `placeholder`. 
but, `value={this.state.quantity}`, was setting the value for each attribute's textfield to what we initialized the state of the PrescriptionAdd page (refer to lines 7-16). for example, quantity would already have a value of 0 and would remove the `placeholder`. 
i have removed the `value={this.state.quantity}` for every attribute to avoid any more confusion... each attribute does not need an initial value because they will be updated by the `onChange` property dependent upon user input.